htcanalyze/main.py

```diff
@@ -44,18 +51,18 @@
                          "errors", "ram-history"]
 
 
-# class to store and change global variables
-class GlobalPlayer(object):
+# this class sets the output and input streams as global variables
+class GlobalStreams(object):
     """handle global variables."""
 
     def __init__(self):
-        """initialize."""
+        """ initialize class without redirecting streams as default """
         self.redirecting_stdout = None
```

Comment:
Rather than duplicating this, couldn't `__init__()` call `reset()`?

Reply:
It can't, because pep8 (I think it's pep8, but in any case it is common practice enough for linters to warn you about it) requires you to define all instance attributes (belonging to `self.`) in `__init__` prior to their use in class methods.